data/json/professions.json

```diff
@@ -1844,6 +1862,7 @@
     "type": "profession",
     "id": "heli_pilot",
     "name": "Helicopter Pilot",
+    "requirement": "achievement_reach_military_bunker",
```

Comment:
Wouldn't this be a problem with the "Last Flight" scenario? I remember that the scenario in question only has this profession as an option...

And I don't think a normal pilot that does not spawn with weapons (or an helicopter if chosen outside the scenario) really needs to be locked behind reaching a military bunker, I feel like this one could be done with no requirement at all.

Reply:
Good eye. Need to remove the requirement from this profession.